### ISSUE TITLE
dark theme interface

### ISSUE THREAD

**Issue by NektoNektovich** [Slion/Fulguris#408]
fulguris has two identical very black themes for what? My eyes hurt from both snow-white and black! 😣 please make the dark theme exactly like the original lightning browser, and please add a setting for the light theme of sites with a dark theme of the browser interface 🤓

**Comment by Slion** [comment]
> fulguris has two identical very black themes for what?

That is indeed questionable and is mostly historical I guess.
One is black, those were designed for OLED screen to save power by turning off as many pixels as possible.
One is dark, more aligned with the dark themes you find today I guess.

> please make the dark theme exactly like the original lightning browser

I'm not sure what that looked like TBH. Though I'm guessing that's close from the early Android dark theme and those were just kinda darg gray if I recall well.

> please add a setting for the light theme of sites with a dark theme of the browser interface 🤓

Good point, not sure that's doable.

> My eyes hurt from both snow-white and black

Sorry to hear that, is the dark theme still too dark for you?

**Comment by NektoNektovich** [comment]
> > fulguris has two identical very black themes for what?
> 
> That is indeed questionable and is mostly historical I guess. One is black, those were designed for OLED screen to save power by turning off as many pixels as possible. One is dark, more aligned with the dark themes you find today I guess.
> 
> > please make the dark theme exactly like the original lightning browser
> 
> I'm not sure what that looked like TBH. Though I'm guessing that's close from the early Android dark theme and those were just kinda darg gray if I recall well.
> 
> > please add a setting for the light theme of sites with a dark theme of the browser interface 🤓
> 
> Good point, not sure that's doable.
> 
> > My eyes hurt from both snow-white and black
> 
> Sorry to hear that, is the dark theme still too dark for you?

I use the fulguris dark theme, not the black amoled one, but I don’t see any serious differences between them, they are both too dark. Nearby I use a browser in 1dm 15.2, which is also based on lightning browser, there is a dark theme that is original and very pleasing to the eyes, there is also a setting for a light theme for sites with a dark browser interface, as well as an original algorithm for hiding the address bar without sticking when scrolling, and you in a vertical orientation, the address bar sticks in a hidden state with any setting, of which there are many, but they work the same, which is inconvenient for me, I can’t set it up for myself🙁 I also wanted to ask for the integration of third-party download managers, like mobile firefox, vivaldi, pure browser, via browser, but I'm glad that although there is a built-in one, the download destination folder of which can be transferred to the memory card along the path Android / data.

**Comment by luckkmaxx** [comment]
Regarding themes, maybe they would be according with new Monet android themes.
Apart of this, regarding people who hurts they eyes on some colors, there is an open source color schemes named [Solarized](https://github.com/altercation/solarized) (dark and light), wich helps exactly on this kind of problem.

**Comment by The-First-King** [comment]
In the Lightning web-browser, the black (for AMOLED) theme worked great. It has been applicable to the web-browser's interface only and not to websites. In Fulguris, when selecting this theme, it impacts website's coloring also. For example, www.samba.org turns into a dark mode. Is it possible to use AMOLED black them without changing the default color of websites.

**Comment by Slion** [comment]
I think it is feature of the website itself. It might be possible the override it but I don't really see the point of having dark UI and light websites.

**Comment by Slion** [comment]
Yes it's very likely that since Lightning is still using an older version of WebKit and Android SDK and theme it would not support web site theming. Therefore web sites will just provide their default theme which is usually the light one. It's not a feature it's a bug 😁

The correct and expected behaviour is for web sites to use the theme currently used by the application which is what Fulguris does at the moment.

There might be ways to select the web site colour scheme independently from the UI colour scheme, we could even possibly make this a domain preference. Though I'm not sure which is the best approach for that.
